spring-integration-hazelcast/README.md

```diff
@@ -200,6 +200,40 @@ Sample definition is as follows :
 **Reference :** http://docs.hazelcast.org/docs/3.4/manual/html-single/hazelcast-documentation.html#continuous-query 
 
 
+## HAZELCAST CLUSTER MONITOR INBOUND CHANNEL ADAPTER 
+
+Hazelcast Cluster Monitor enables to listen to the modifications performed on cluster. Hazelcast Cluster Monitor Inbound Channel Adapter is an event-driven inbound channel adapter and listens related Membership, Distributed Object, Migration, Lifecycle and Client events. Its basic definition is as follows : 
```

Comment:
or `event-driven`, or `inbound`, but not both :smile: 
Actually we call that like `message-driven`.